kalite/distributed/backup.py

```diff
@@ -0,0 +1,36 @@
+import dbbackup
+import datetime
+import os
+import sys
+import time
+from os.path import expanduser
```

Comment:
@benjaoming We'll be needing only os.path.expanduser and os.path.join. Should we still import "os" completely or should I replace import.os with two imports: expanduser/path.join?